CHANGELOG.md

```diff
@@ -6,7 +6,177 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+  ```text
+  CREATE INDEX files_granule_cumulus_id_index ON files (granule_cumulus_id);
+  ```
+
+  You should see a response like:
+
+  ```text
+  CREATE INDEX
+  ```
+
+  and can verify the index `files_granule_cumulus_id_index` was created:
+
+  ```text
+  => select * from pg_indexes where tablename = 'files';
+  schemaname | tablename |           indexname            | tablespace |                                           indexdef
+   ------------+-----------+--------------------------------+------------+----------------------------------------------------------------------------------------------
+   public     | files     | files_pkey                     |            | CREATE UNIQUE INDEX files_pkey ON public.files USING btree (cumulus_id)
+   public     | files     | files_bucket_key_unique        |            | CREATE UNIQUE INDEX files_bucket_key_unique ON public.files USING btree (bucket, key)
+   public     | files     | files_granule_cumulus_id_index |            | CREATE INDEX files_granule_cumulus_id_index ON public.files USING btree (granule_cumulus_id)
+  (3 rows)
+  ```
+
+  - Once this is complete, you may deploy this version of Cumulus as you
+    normally would.
+  **If you are unable to stop ingest for the above procedure** *and* cannot
+  migrate with deployment, you may be able to manually create the index while
+  writes are ongoing using postgres's `CONCURRENTLY` option for `CREATE INDEX`.
+  This can have significant impacts on CPU/write IO, particularly if you are
+  already using a significant amount of your cluster resources, and may result
+  in failed writes or an unexpected index/database state.
+
+  PostgreSQL's
+  [documentation](https://www.postgresql.org/docs/10/sql-createindex.html#SQL-CREATEINDEX-CONCURRENTLY)
+  provides more information on this option.   Please be aware it is
+  **unsupported** by Cumulus at this time, so community members that opt to go
+  this route should proceed with caution.
+
+  -----
+
+### Notable changes
+
+- **CUMULUS-2962**
+  - Re-added database structural migration to `files` table to add an index on `granule_cumulus_id`
+- **CUMULUS-2929**
+  - Updated `move-granule` task to check the optional collection configuration parameter
+    `meta.granuleMetadataFileExtension` to determine the granule metadata file.
+    If none is specified, the granule CMR metadata or ISO metadata file is used.
+
+### Changed
+
+- Updated Moment.js package to 2.29.4 to address security vulnerability
+- **CUMULUS-2967**
+  - Added fix example/spec/helpers/Provider that doesn't fail deletion 404 in
+    case of deletion race conditions
+
+### Fixed
+
+- **CUMULUS-2863**
+  - Fixed `@cumulus/api` `validateAndUpdateSqsRule` method to allow 0 retries and 0 visibilityTimeout
+    in rule's meta.
+
+- **CUMULUS-2959**
+  - Fixed `@cumulus/api` `granules` module to convert numeric productVolume to string
+    when an old granule record is retrieved from DynamoDB
+
+## [v13.0.0] 2022-06-13
```

Comment:
This doesn't seem right....

Reply:
https://github.com/nasa/cumulus/pull/3033/commits/d71f99e7d538ad6dd07d6a95983af9f02b14637e 😞